### PR TITLE
Changed `backends.default` to `backends.allowed`.

### PR DESCRIPTION
**Pull request by geoffjentry**
Removed dead code related to configs such as the old `backend.backend`.
Updated unencrypted configuration files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1432,15 +1430,15 @@ Sample usage :
 
 ```wdl
 task sparkjob_with_yarn_cluster {
-        .....
+        #.....
```
Comment (mcovarr) [inline README.md]: why the octothorped ellipses?

**Comment by mcovarr** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -551,14 +551,14 @@ Backends are specified in the `backend` configuration block under `providers`.
 
 ```hocon
 backend {
-  default = "Local"
+  allowed = ["Local"]
```
Comment (cjllanwarne) [inline README.md]: Pure ToL, the first one being the default struck me as non-intuitive. For whatever reason it seems less odd to me if it was called `enabled` or `active`... (shrug)

**Comment by cjllanwarne** [comment]
👍

File: README.md (hunk 1 above)
Comment (geoffjentry) [inline README.md]: I suspect because it's not syntactically correct but it's to demonstrate that one could have more stuff?

File: README.md (hunk 2 above)
Comment (geoffjentry) [inline README.md]: I like enabled

File: .travis.yml [hunk 3]
```diff
@@ -16,7 +16,7 @@ before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 before_install:
-  - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
+  - openssl aes-256-cbc -K "$encrypted_5ffc1dec8cd8_key" -iv "$encrypted_5ffc1dec8cd8_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
```
Comment (cjllanwarne) [inline .travis.yml]: This isn't secret keymat, right...?

Comment (geoffjentry) [inline .travis.yml]: Huh?

Comment (cjllanwarne) [inline .travis.yml]: I think it's ok. Saw a bunch of hex and the word "encrypted" and wanted to make sure it wasn't accidentally leaking key material. But actually it's just a variable name.

Comment (geoffjentry) [inline .travis.yml]: It's the encrypted crap that Travis decodes

**Pull request closed without merging**